### PR TITLE
Fix comment on after script to copy include files into correct path

### PR DESCRIPTION
**Pull request by rverrips**
The include files are copied into the directory one level up from the actual target path/folder
Similar incorrect path used for launching codeeditor

### CODE REVIEW

File: lambo [hunk 1]
```diff
@@ -285,7 +285,7 @@ cd "$PROJECTNAME" || exit
 
 if [[ "$CODEEDITOR" != "" ]]; then
     if ! isterminaleditor; then
-       $CODEEDITOR .
+       $CODEEDITOR $PROJECTPATH/$PROJECTNAME
```
Comment (mattstauffer) [inline lambo]: This is unnecessary and actually makes it not work when I test it.

File: lambo [hunk 2]
```diff
@@ -113,7 +113,7 @@ makeafter()
 
 # To copy standard files to new lambo project place them in ~/.lambo/includes directory.
 # echo "Copying Include Files"
-# cp -R ~/.lambo/includes/ $PROJECTPATH
+# cp -R ~/.lambo/includes/ $PROJECTPATH/$PROJECTNAME
```
Comment (mattstauffer) [inline lambo]: This makes it copy into a subdirectory, not the appropriate directory.

File: lambo [hunk 3]
```diff
@@ -317,16 +317,21 @@ perlCommands=(
     's/(DB_USERNAME=)(.*)/\1root/g'
     's/(DB_PASSWORD=)(.*)/\1/g'
     "s/(APP_URL=)(.*)/\1http:\/\/$PROJECTNAME.$TLD/g"
+    "s/(APP_NAME=)(.*)/\1$PROJECTNAME/g"
```
Comment (mattstauffer) [inline lambo]: 👌🏻

File: lambo [hunk 4]
```diff
@@ -350,7 +355,7 @@ fi
 
 if [[ "$CODEEDITOR" != "" ]]; then
     if isterminaleditor; then
-       $CODEEDITOR .
+       $CODEEDITOR $PROJECTPATH/$PROJECTNAME
```
Comment (mattstauffer) [inline lambo]: This doens't work; it actually keeps my IDE (Sublime Text) from opening it correctly.

**Comment by mattstauffer** [comment]
@rverrips Is this something you're thinking about coming back to?

**Comment by rverrips** [comment]
Apologies @mattstauffer - totally dropped off my radar.  I’ll revisit it this week and update you in the next few days.

**Comment by mattstauffer** [comment]
@rverrips No worries--I've dropped the ball a *lot* on Lambo lately! :)

**Comment by rverrips** [comment]
I relooked at this - Apologies for taking so long.
In reading through my commit notes the plan was to allow lambo to be a "global" script, i.e. could be run from anywhere in the console, not just from the ~/Sites or ~/Code folder, and wouldn't require you first "cd ~/Sites". 
To achieve that needed to set $PROJECTPATH in the Lambo config to whatever your preference (or use the path you're running from as default) 
I'll reimplement this in another PR with better documentation / explanation.

**Pull request closed without merging**